editor/src/components/canvas/canvas-strategies/strategies/group-like-helpers.ts

```diff
@@ -26,6 +26,19 @@ export function retargetStrategyToChildrenOfContentAffectingElements(
   )
 }
 
+export function retargetStrategyToTopMostGroupLikeElement(
```

Comment:
Worth having a test to cover what this does?

Reply:
see test I un'xited in editor/src/components/canvas/canvas-strategies/strategies/flex-reorder-strategy.spec.browser2.tsx

Reply:
I was thinking more a unit test for this, but that exercises it at the least.

Reply:
right. a unit test if probably warranted